cypress/integration/Session.feature

```diff
@@ -0,0 +1,34 @@
+@Session
+Feature: Session
+
+    I want to be notified when my session is about to expire
+
+    Background:
+        Given I ignore application exceptions
+
+    @focus
+    Scenario: Session timeout pop-up displays after inactivity and keeps focus (LPAL-245)
```

Comment:
tag this test so it only runs in CI

Reply:
After modifying how this works, it's now possible to modify the session timeout via a query string variable, so this tag is no longer necessary.